gnovm/stdlibs/crypto/sha256/sha256.go

```diff
@@ -0,0 +1,7 @@
+package sha256
+
+import "crypto/sha256"
+
+func Sum256(data []byte) [32]byte {
+	return sha256.Sum256(data)
+}
```

Comment:
Do we need to expose every method for any Go standard library public method?

Reply:
Well, the goal is not to start implementing standard libraries by implementing all of their code as a native mapping. The "native" part should only be what is strictly necessary or makes sense for performance; including Sha256 in this case, as it is highly optimised and the native performance will greatly outperform a sha256 implementation done in pure gno.